README.md

```diff
@@ -120,48 +129,175 @@ list, I am an expert in none of these technologies).
 ### Go back to the roots to became a real software engineer
 
 1. It is not too late to learn Computer Science. Here is
-   [a program I plan to do one day](https://teachyourselfcs.com/)
+   [a program I plan to do one day].
 
 ### Try something different in back-end
 
-1. Learn a functional language. Elixir and Phoenix (the most popular framework
-   for Elixir) are great choices for Rubyist as Elixir was written by a former
-   Rails core contributor.
+1. Learn a functional language. [Elixir] and [Phoenix] (the most popular
+   framework for Elixir) are great choices for Rubyist as Elixir was written by
+   a former Rails core contributor.
 
 ### Introduce yourself to data-science
 
-1. I am currently following the
-   [Data Analyst Nanodegree on Udacity](https://www.udacity.com/course/data-analyst-nanodegree--nd002).
-   It have been great so far!
+1. I am currently following the [Data Analyst Nanodegree on Udacity]. It have
+   been great so far!
 
 ### Focus on front-end
 
-1. Improve your CSS skills
-2. Learn a trendy JS library (React or Vue.js for example). Good resources
-   include Wes Bos stuffs or EggHead tutorial. I personnaly have a EggHead
-   subscription as it allows to quickly get introduced to new topics before
-   going deeper by yourself if it catches your interest.
+1. Learn how to structure properly your vanilla CSS with [BEM]: for reusable
+   components & a real naming convention for your next big projects.
+1. Then, show your skills to the entire world on [Codepen] by sharing some
+   [next level Japanese Sushi Toggle Button] ! :yum:
+1. Of course, your Codepen profile can be written in something a bit more
+   advanced if you wish. Why not [Stylus] (some cool features and ponctuation
+   free) and [Haml] (_a bit_ more readable HTML) ?
+1. Bootstrap is nice but you want to discover other CSS frameworks ? There are
+   plenty of them, like [Materialize], [Bulma] or [Carbon Design System].
+1. Want to know what Bootstrap does behind the scenes in it's 4th version ? Wes
+   has a free [Flexbox] lesson just for you ! You'll see that it's helps a lot
+   for some homemade responsive Webdesign.
+1. CSS Grid is another cool feature released not so long ago. We don't have any
+   goods links here but feel free to pull request one ! :smirk:
+1. Tell me something, do you like the animations of this [studio] ? It's using
+   [Greensock], part of it is free and it allows you to make some awesome
+   design effects. [Petr Tichy] has some free lessons on it !
 
 ### Become a mobile developer
 
-1. Learn React Native (and plug it to a Rails API) to build native applications
-   on mobile
+1. Learn a trendy JS library (ReactJS or Vue.js for example). One of the best
```

Comment:
Not linked to mobile dev. This is front end stuff.

Reply:
The thing is that, mobile dev is pretty just React Native & ReactJS is a bigger part in our ressources. I'll just rename `Become a mobile developer` to `Discover React` I guess. ✋

Reply:
I would keep the "Focus on front-end section" (it already talks about React) and would just put React Native resources in the "Mobile" section. However, it is indeed a great indead to let people know React is a requirement. So we can write: `Be careful, before going through these resources to learn React Native, you should have a good understanding of React`